### PR TITLE
fix: append cap readme

### PR DESCRIPTION
**Pull request by longieirl**
Append root README for CAP projects.
- some minor updates to existing README's

### CODE REVIEW

File: cap/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+2. [Standalone Approuter](/cap-fiori-mta-standalone/README.md) with an inmemory database
+
+Both CAP project were generated using the steps outlined in this [blog post](https://blogs.sap.com/2022/02/10/build-and-deploy-a-cap-project-node.js-api-with-a-sap-fiori-elements-ui-and-a-managed-approuter-configuration/) selecting the appropiate approuter configuration as the HTML5 application runtime.
+
+### Known Issues
+There are no known issues relating to the resources included in this repository. Please refer to **How to obtain support** for any issue you encounter.
+
+### How to Obtain Support
+- Option 1 : Users can log in and [submit their question](https://answers.sap.com/questions/ask.html) in the **SAP Community**. Please select the tag "SAP Fiori tools".
+- Option 2 : Use this option ONLY when you encounter any issues using the sample project with [SAP Developer tutorials](https://developers.sap.com/group.fiori-tools-lrop.html), please report it [here](https://github.com/SAPDocuments/Tutorials/issues/new).
```
Comment (theStiggySpd) [inline cap/README.md:22]: hi @longieirl We can get rid of this support option.

**Review by theStiggySpd — CHANGES_REQUESTED** [review]
Change in support options requested

**Comment by longieirl** [comment]
@theStiggySpd ready whenever you are. Thanks.

**Review by theStiggySpd — APPROVED** [review]
looks good. Thanks much

**Pull request merged**